api/api.proto

```diff
@@ -359,6 +359,10 @@ enum ProxyProtocol {
     PROXY_PROTOCOL_S3 = 2;
     // PXD indicates that the external data source is a Portworx block volume.
     PROXY_PROTOCOL_PXD = 3;
+    // PURE_BLOCK indicates that the external data source is a pure_block volume.
+    PROXY_PROTOCOL_PURE_BLOCK = 4;
+    // PURE_FILE indicates that the external data source is a pure_file volume.
+    PROXY_PROTOCOL_PURE_FILE = 5;
```

Comment:
For these changes, you will need to bump the API minor version in `api.proto`. Found here
https://github.com/libopenstorage/openstorage/blob/master/api/api.proto#L4863

After bumping the minor version, please add some release notes here (new version will be 116)
https://github.com/libopenstorage/openstorage/blob/master/SDK_CHANGELOG.md